lib/spree/core/calculated_adjustments.rb

```diff
@@ -26,25 +26,24 @@ def calculator_type=(calculator_type)
           #   (which is any class that has_many :adjustments) and sets amount based on the
           #   calculator as applied to the given calculable (Order, LineItems[], Shipment, etc.)
           # By default the adjustment will not be considered mandatory
-          def create_adjustment(label, target, calculable, mandatory = false, state = "closed")
-            amount = compute_amount(calculable)
+          def create_adjustment(label, adjustable, mandatory = false, state = "closed")
+            amount = compute_amount(adjustable)
             return if amount.zero? && !mandatory
 
             adjustment_attributes = {
               amount: amount,
-              source: calculable,
               originator: self,
-              order: order_object_for(target),
+              order: order_object_for(adjustable),
               label: label,
               mandatory: mandatory,
               state: state,
-              included: tax_included?(self, target)
+              included: tax_included?(self, adjustable)
             }
 
-            if target.respond_to?(:adjustments)
-              target.adjustments.create(adjustment_attributes)
+            if adjustable.respond_to?(:adjustments)
+              adjustable.adjustments.create(adjustment_attributes)
             else
-              target.create_adjustment(adjustment_attributes)
+              adjustable.create_adjustment(adjustment_attributes)
```

Comment:
this is not relevant to this PR but I suggest we make all adjustable implement `create_adjusment` however suits them best. Either using an `adjustements` association or something else. This would decouple this class from these creation details.

I feel all this code suffers too much from a lack of [Tell, Don't Ask](https://www.martinfowler.com/bliki/TellDontAsk.html).